src/neptune/common/backends/utils.py

```diff
@@ -100,6 +101,7 @@ def wrapper(*args, **kwargs):
                 HTTPTooManyRequests,
                 HTTPInternalServerError,
                 NewConnectionError,
+                ChunkedEncodingError,
```

Comment:
How about retrying only once like in case of SSL error?

Reply:
Also some very explanatory comment would be welcome.

Reply:
I suppose, that client closes the connection before forking and after resuming - client doesn't receive a response from server.

This error is similar to `HTTPRequestTimeout or BravadoConnectionError` - and we can retry this request.

Reply:
It's hard to believe that it throws some very random exception when the connection is just closed. Can we test this hypothesis?